src/frontends/pytorch/src/op_table.cpp

```diff
@@ -1082,6 +1082,7 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_fx() {
         {"aten.zeros.names", op::translate_zeros_fx},
         {"aten.zeros_like.default", op::translate_zeros_like_fx},
         {"get_attr", op::translate_constant},
+        {"scalar_tensor.default", op::translate_scalar_tensor_fx},
         {"torchvision.deform_conv2d.default", op::translate_deform_conv},
```

Comment:
please add layer tests for both torch_export and torch_fx scenarious

Reply:
@mvafin, "scalar_tensor" seems to be added as part of the commit below. Does any of the tests already cover it?
https://github.com/openvinotoolkit/openvino/commit/d018779adda8067952e020e9d946b11924d00fdb

Reply:
scalar_tensor test added